tests/test_particle_track.py

```diff
@@ -133,6 +133,17 @@ def test_exact_locations_overflow():
     assert len(all_walk_data['xinds']) == num_ps
     assert len(all_walk_data['yinds']) == num_ps
 
+def test_exact_locations_multidims():
+    """Test case where seed locs given as multi-dimensional arrays."""
+    num_ps = 5
+    particle = particle_track.Particles(params)
+    x_seed = np.array(((0, 0), (1, 1)))
+    y_seed = np.array(((0, 0), (1, 1)))
```

Comment:
It tries to flatten them ([from here](https://github.com/passaH2O/dorado/blob/f1ec04e2fe451751906444b1e807c967ab5b952c/dorado/particle_track.py#L708)) however we didn't test that it actually *did* flatten them and still function, so that was the aim of this test.